src/db/clients/ClientUtils.ts

```diff
@@ -0,0 +1,16 @@
+import DBProvider from '../entity/enum';
+import { ConnectionModelType } from '../models/ConnectionModels';
+import PgClient from './PgClient';
+
+const getRightClient = (model: ConnectionModelType, database?: string) => {
+  switch (model.type) {
+    case DBProvider.PostgreSQL: {
+      return new PgClient(model, database);
+    }
+    default: {
```

Comment:
This default case is fine for now, but when we add more databases we probably want to throw an error or something if none of the cases are matched. Based on the current design it's not even possible to not select a dbprovider in the enum.

Reply:
Oh yeah, that's a good catch I had that in there for now but we can remove that for sure